SpywareFilter/sections/mobile_whitelist.txt

```diff
@@ -2,6 +2,9 @@
 ! Exception rules that fix broken mobile apps and mobile websites
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/81513
+@@||google-analytics.com^$script,domain=m.dcinside.com|mupload.dcinside.com
```

Comment:
What about to make this rule more narrow? For example these 2 rules works good on my device, please check:
```
@@||google-analytics.com/analytics.js^$script,domain=m.dcinside.com|mupload.dcinside.com
@@||ssl.google-analytics.com/ga.js^$script,domain=m.dcinside.com|mupload.dcinside.com
```

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/81571/commits/754569f133cf9f63ab7ea5870238177620823d68

Reply:
@piquark6046
Did you check them?

Reply:
@Sergey-Lyapin 
Yes. On AG browser extension, they work.

Reply:
Ok, thank you